package.json

```diff
@@ -65,7 +65,7 @@
     "intersection-observer": "^0.12.0",
     "playkit-js-providers": "https://github.com/kaltura/playkit-js-providers.git#v2.32.0",
     "proxy-polyfill": "^0.3.0",
-    "shaka-player": "3.0.13"
```

Comment:
no changes are required in the adapter ?

Reply:
I did in dash repo of course - https://github.com/kaltura/playkit-js-dash/pull/178